flyway/sql/reference_data/R__1_create_superuser.sql

```diff
@@ -0,0 +1,33 @@
+-- upsert a bootstrap / default superuser account if and only if a variable has
+-- been passed into the `flyway migrate` script
+--
+-- WARNING: any time this script is run with a new value for
+-- `superuserDefaultPassword`, the INSERT will be attempted.
+INSERT INTO
+    public.auth_user (
+        password,
+        is_superuser,
+        username,
+        email,
+        first_name,
+        last_name,
+        is_staff,
+        is_active,
+        date_joined
+    )
+SELECT
+    '${superuserDefaultPassword}',
```

Comment:
I'm assuming this comes from `FLYWAY_PLACEHOLDERS_superuserDefaultPassword` but `FLYWAY_PLACEHOLDERS_` is stripped from the key?